setup.py

```diff
@@ -6,20 +6,18 @@
 ROOT_DIR = os.path.dirname(__file__)
 SOURCE_DIR = os.path.join(ROOT_DIR)
 
-requirements = ["nose==1.3.0", "splinter>=0.4.9", "lettuce>=0.2.19"]
+requirements = ["nose", "splinter", "lettuce"]
 try: import argparse
 except ImportError: requirements.append('argparse')
 
 setup(
     name="salad",
     description="A nice mix of great BDD ingredients",
-    author="Lieferheld QA",
-    author_email="qa@deliveryhero.com",
-    url="https://github.com/beanqueen/salad",
+    author="Jana Rekittke",
+    author_email="jana@rekittke.name ",
```

Comment:
The author and author_email should actually stay (I must have missed this in previous PRs) as

```python
author="Steven Skoczen",
author_email="steven@inkandfeet.com",
```

Please note that this is authorship of the entire project, and who people should contact for support, not just this PR.